README.rst

```diff
@@ -135,11 +188,46 @@ is set. When doing so, it will override any parameters set in the
             #     'download_slot': 'quotes.toscrape.com'
             # }
 
-The raw Zyte Data API response can be accessed via the ``raw_api_response`` attribute
-of the response object. Note that such responses are of ``ZyteAPIResponse`` and
-``ZyteAPITextResponse`` types, which are respectively subclasses of ``scrapy.http.Response``
-and ``scrapy.http.TextResponse``. Such classes are needed to hold the raw Zyte Data API
-responses.
+There is a shortcut, in case a request uses the same parameters as
+defined in ``ZYTE_API_DEFAULT_PARAMS`` setting, without any further
+customization - ``zyte_api`` meta key can be set to True:
```

Comment:
```suggestion
customization - ``zyte_api`` meta key can be set to ``True`` (or even ``{}``):
```

Reply:
I believe an explicit mention of the `{}` would be a key tip for users that needs to programmatically set their params like:
```python
meta = {"zyte_api": {}}
if need_geolocation:
    meta["zyte_api"]["geolocation"] = "FR"
yield scrapy.Request(url, self.some_method, meta=meta)
```